_maps/map_files/IceBoxStation/IcemoonUnderground_Above_skyrat.dmm

```diff
@@ -16272,6 +16422,17 @@
 /obj/item/crowbar,
 /turf/open/floor/iron,
 /area/service/chapel)
+"VX" = (
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden/layer4,
+/obj/machinery/vending/dorms{
+	desc = "A vending machine with various toys. Not for the faint of heart. This one seems to have a reduced inventory.";
+	name = "NanoTrasen Approved LustWish";
```

Comment:
May want to make this its own type to allow for use of it outside this map without copious varediting

Reply:
Hm sure makes sense